### DISCUSSION TITLE
Is there a way to save a session and save a dashboard?

### DISCUSSION THREAD

**Discussion by pmanandhar** [atoti/atoti#509] [Q&A] [answered]
I tried using  following user_content_storage for creating a session.

```py
session = tt.create_session(config = {"user_content_storage":"./content"})
```

It works as long as the session is open, but once we close the session, the dashboard content is lost.
So, is there a way to save this session and restore the session as well as the content of the dashboard saved?

**Comment by tibdex** [comment, marked as answer]
Your code seems correct. Did you **Save** your dashboard before closing the session and restarting it?

<img width="1028" alt="Screen Shot 2022-03-09 at 10 53 44" src="https://user-images.githubusercontent.com/6574550/157417557-6deeff52-9120-44c0-9ce8-f5fa6a94db23.png">

**Comment by pmanandhar** [comment]
Yes, I did save the dashboard.
Can you help me by letting me know how to restart the session?

**Reply by bloussou** [reply]
You just need to re-execute the cell if you are in a notebook, or re-execute the script if you have written a script

**Reply by pmanandhar** [reply]
Thanks

**Comment by AKSTC** [comment]
Hi. is this answer still valid? I'm using atoti 0.8.2 and it is getting the below error.

AttributeError: module 'atoti' has no attribute 'create_session'

**Comment by pmanandhar** [comment]
this syntax for creating session has been obsolete.
The new syntax looks like this:
session = tt.Session()

or if you want to add additional parameters then:

session = tt.Session(
  user_content_storage="./content",
    java_options=[
      "-Xmx48g",
  ],
)

**Reply by AKSTC** [reply]
Thank you for your prompt support. I was struggling to save my dashboard on my local PC and your code worked...
